packages/java/plugin-base/src/main/java/dev/hilla/plugin/base/HillaAppInitUtility.java

```diff
@@ -15,15 +15,15 @@ public class HillaAppInitUtility {
     private static final Logger LOGGER = LoggerFactory
             .getLogger(HillaAppInitUtility.class);
 
-    private static final String REACT_SKELETON = "https://github.com/vaadin/skeleton-starter-hilla-react/archive/refs/heads/v2.1.zip";
+    private static final String REACT_SKELETON = "https://github.com/vaadin/skeleton-starter-hilla-react/archive/refs/heads/v2.5.zip";
```

Comment:
Wouldn't it be better to use e.g. `Platform.getHillaVersion()` so that the value doesn't have to be hardcoded?

Reply:
I agree with both of you, but this is not the good time to play with version-related stuff, see #1904 and #1905. Hopefully we'll include some better version tracking in files like this one when changing version handling.